### PR TITLE
Release prep for v1.0.0

### PR DESCRIPTION
**Pull request by genebean**
Note that I included a release summary as part of the changelog by way of #4. It mostly reflects what is at the bottom of the readme, but I wanted to call it out for transparency.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-falco",
-  "version": "0.4.0",
+  "version": "1.0.0",
   "author": "puppet",
```
Comment (bastelfreak) [inline metadata.json:4]: please update this to `Vox Pupuli`

**Comment by genebean** [comment]
Switching this back to draft for a moment, I have discovered a bug in the default config.

**Comment by genebean** [comment]
https://github.com/voxpupuli/puppet-falco/pull/6 fixes the issue I observed.

File: metadata.json (hunk 1 above)
Comment (genebean) [reply metadata.json]: Upated

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**